pkg/utils/pod/provisioning.go

```diff
@@ -12,22 +12,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package scheduling
+package pod
```

Comment:
Be careful to avoid package names that conflict with commonly used concepts elsewhere in the codebase. Almost certainly we're going to have a conflict here where pod is both a variable and package name, forcing you to rename the package upon import.